include/umf/memory_provider_ops.h

```diff
@@ -125,10 +125,11 @@ typedef struct umf_memory_provider_ipc_ops_t {
     /// @brief Closes an IPC memory handle.
     /// @param provider pointer to the memory provider.
     /// @param ptr pointer to the memory retrieved with open_ipc_handle function.
+    /// @param size size of the memory address range.
```

Comment:
"`size size`" -> `size`

Reply:
but the first `size` is an argument name and the second on is part of the comment. 

We have the same comments for other methods with `size` parameter.

Reply:
ah :-) OK, I see now